core/src/scene/sceneLoader.cpp

```diff
@@ -98,9 +98,10 @@ bool SceneLoader::applyUpdates(const std::shared_ptr<Platform>& platform, Scene&
             try {
                 // Dummy node to trigger YAML exception on YAML syntax errors
                 auto parse = YAML::Load(update.path);
-                Node node = YamlPath(update.path).get(root);
+                bool validPath = false;
+                auto node = YamlPath(update.path).get(root, validPath);
 
-                if (node && node.Scalar().empty() && node != root) {
+                if (!validPath || (node && node.Scalar().empty() && node != root)) {
```

Comment:
Discussed on slack, this test is similar to `validPath` and can be removed. For convenience, the interface to test `YamlPath` validity could be changed for that:

```c++
bool YamlPath::get(YAML::Node node, Node* node);
```
so it's used as such:
```c++
Node n;
if (YamlPath(update.path, &n).get()) {
   // valid
} else {
  // invalid
}```